src/curd.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable import/no-cycle */
+import { todoList, populateList } from './index.js';
+import bin from './assets/recyclebin.png';
+
+// Function to save the todoList to local storage
+export const saveTodoListToLocalStorage = () => {
+  localStorage.setItem('todoList', JSON.stringify(todoList));
+};
+
```

Comment:
- [ ] Hello, When I refresh the page, all the data in your todo App are removed. Normally, these data don't need to disappear unless I delete them. Can you fix the issue?
**NB**: I provided two pictures below: stage 1 is when the page is not refreshed yet. Stage 2 is when I refresh the page. I hope this will help you understand.

- [ ] Don't disable the linter, the linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs
- 
![Capture](https://github.com/umarahgul/webpackSetup/assets/63972670/ea86c8e4-0941-4364-a2b6-849b00e25fe3)
-

![Capture1](https://github.com/umarahgul/webpackSetup/assets/63972670/b6a9ca35-3902-452f-97b0-1ce20293ee97)